tools/travis/scan.sh

```diff
@@ -41,7 +41,7 @@ cd $ROOTDIR
 TERM=dumb ./gradlew checkScalafmtAll
 
 # lint tests to all be actually runnable
-MISSING_TESTS=$(grep -rL "RunWith" --include="*Tests.scala" tests)
+MISSING_TESTS=$(grep -rL "RunWith" --include="*Tests.scala" tests || true)
```

Comment:
seems ubuntu 20.04 will return an error for this command if `grep` matched nothing